GameData/RealismOverhaul/RO_SuggestedMods/HabTech2/RO_HabTech2_TACLS.cfg

```diff
@@ -0,0 +1,1606 @@
+
+
+@PART[ht2_MPLM]:FOR[RealismOverhaul]:NEEDS[TacLifeSupport]
+{
+	@MODULE[ModuleFuelTanks]
+	{
+		TANK
+		{
+			name = CarbonDioxide
+			amount = 0
+			maxAmount = 4000
+		}
+		TANK
+		{
+			name = Oxygen
+			amount = 4000
+			maxAmount = 4000
+		}
+		TANK
+		{
+			name = Waste
+			amount = 0
+			maxAmount = 10
+		}
+		TANK
+		{
+			name = WasteWater
+			amount = 0
+			maxAmount = 60
+		}
+		TANK
+		{
+			name = Food
+			amount = 100
+			maxAmount = 100
+		}
+		TANK
+		{
+			name = Water
+			amount = 75
+			maxAmount = 75
+		}
+		TANK
+		{
+			name = LithiumHydroxide
+			amount = 200
+			maxAmount = 200
+		}
+	}
+
+	MODULE
+	{
+		name = TacGenericConverter
+		converterName = Oxygen Generator (Electrolysis)
+		StartActionName = Start Oxygen Generator (Electrolysis)
+		StopActionName = Stop Oxygen Generator (Electrolysis)
+		tag = Life Support
+		GeneratesHeat = False
+		UseSpecialistBonus = True
+		SpecialistEfficiencyFactor = 0.2
+		SpecialistBonusBase = 0.05
+		ExperienceEffect = ConverterSkill
+		EfficiencyBonus = 1
+		conversionRate = 7.0
+
+		INPUT_RESOURCE
+		{
+			ResourceName = Water
+			Ratio = 0.0000053129
+		}
+
+		INPUT_RESOURCE
+		{
+			ResourceName = ElectricCharge
+			Ratio = 0.5
+		}
+
+		OUTPUT_RESOURCE
+		{
+			ResourceName = Hydrogen
+			Ratio = 0.006612957
+			DumpExcess = True
+		}
+
+		OUTPUT_RESOURCE
+		{
+			ResourceName = Oxygen
+			Ratio = 0.003116887
+			DumpExcess = True
+		}
+	}
+	MODULE
+	{
+		name = TacGenericConverter
+		converterName = CO2 Filter
+		StartActionName = Start CO2 Filter
+		StopActionName = Stop CO2 Filter
+		tag = Life Support
+		GeneratesHeat = False
+		UseSpecialistBonus = True
+		SpecialistEfficiencyFactor = 0.2
+		SpecialistBonusBase = 0.05
+		ExperienceEffect = ConverterSkill
+		EfficiencyBonus = 1
+		conversionRate = 7.0
+
+		INPUT_RESOURCE
+		{
+			ResourceName = CarbonDioxide
+			Ratio = 0.00625
+		}
+
+		INPUT_RESOURCE
+		{
+			ResourceName = ElectricCharge
+			Ratio = 0.01
+		}
+
+		INPUT_RESOURCE
+		{
+			ResourceName = LithiumHydroxide
+			Ratio = 0.0000085683
+		}
+
+		OUTPUT_RESOURCE
+		{
+			ResourceName = WasteWater
+			Ratio = 0.0000046828
+			DumpExcess = True
+		}
+
+		OUTPUT_RESOURCE
+		{
+			ResourceName = Waste
+			Ratio = 0.0000257297
+			DumpExcess = False
+		}
+	}
+	MODULE
+	{
+		name = TacGenericConverter
+		converterName = Water Recycling System
+		StartActionName = Start Water Recycling System
+		StopActionName = Stop Water Recycling System
+		tag = Life Support
+		GeneratesHeat = False
+		UseSpecialistBonus = True
+		SpecialistEfficiencyFactor = 0.2
+		SpecialistBonusBase = 0.05
+		ExperienceEffect = ConverterSkill
+		EfficiencyBonus = 1
+		conversionRate = 7.0
```

Comment:
As example here, I think only "conversionRate" would change?  So I'd look for a tag "AddTACLSWaterRecycler" and also a key "TACLSWaterRecyclerConversionRate" that would be set by several of the parts, and then added in via a patch, like how we do with the fuels in the RF TANK_DEFINITION nodes.